django_startr/views.py

```
@@ -0,0 +1,173 @@
+            user.save()
+        return user
+
+def register_view(request):
+    """View for registering a new user"""
+    if request.user.is_authenticated:
+        return redirect('accounts:dashboard')
+        
+    if request.method == 'POST':
+        form = UserRegistrationForm(request.POST)
+        if form.is_valid():
+            user = form.save()
```

Ruff (0.8.2)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)